### DISCUSSION TITLE
下载最新版win11提醒有病毒，直接删除文件

### DISCUSSION THREAD

**Discussion by shuiliu11** [XTLS/Xray-core#4091] [Q&A]
Xray-windows-64.zip这个版本，其它没有测试。

**Comment by wy15** [comment]
<img width="1116" alt="image" src="https://github.com/user-attachments/assets/6379e97d-3ddb-4a36-a6fc-328ebf36881c">
校验下你下载的文件吧。

**Reply by sincereliu** [reply]
校验过了。没有问题。windows11会杀掉xray-windows-64.zip压缩包。
其他的版本，比如Xray-win7-64.zip，没有上述问题。

**Comment by shuiliu11** [comment]
![Screenshot 2024-12-01 133630](https://github.com/user-attachments/assets/b728ece7-f2e8-41bb-a03b-5ddad0d7d596)
早上下载的，连续下载2次都被隔离，我现在重新再下载一次，正常了。

**Reply by sincereliu** [reply]
好奇怪，我也是，昨天下载就会被杀。刚才实验下载了一下，没有问题了。

**Comment by sincereliu** [comment]
我也遇到这个情况了。

**Comment by RPRX** [comment]
~~谁去提交个误报纠正~~
